layers/+tools/shell/README.org

```diff
@@ -221,12 +221,14 @@ in the current buffer instead of a popup.
 
 ** Multi-term
 
-| Key binding | Description                  |
-|-------------+------------------------------|
-| ~SPC m c~   | create a new multi-term      |
-| ~SPC m n~   | go to next multi-term        |
-| ~SPC m p~   | go to previous multi-term    |
-| ~SPC p $ t~ | run multi-term shell in root |
+| Key binding            | Description                    |
+|------------------------+--------------------------------|
+| ~SPC m c~              | create a new multi-term        |
+| ~SPC m n~              | go to next multi-term          |
+| ~SPC m N~ or ~SPC m p~ | go to previous multi-term      |
+| ~SPC p $ t~            | run multi-term shell in root   |
+| ~SPC m l~              | switch multi-term to line mode |
+| ~SPC m C               | switch multi-term char mode    |
```

Comment:
Missing `~` (tilde) after `~SPC m C`. The corrected line looks like this:
```
| ~SPC m C~               | switch multi-term char mode    |
```

It was part of the previous commits, but I also missed it.

The issue was discovered by clicking on the `View file` button (which leads to: https://github.com/syl20bnr/spacemacs/blob/5ccb08b3ed2221624a599c244bf4ead7607327fa/layers/%2Btools/shell/README.org) near the top right corner of the diff container for the file ` layers/+tools/shell/README.org`.

On the `Files changed` tab (https://github.com/syl20bnr/spacemacs/pull/12130/files)

![chrome_2019-04-14_12-31-12](https://user-images.githubusercontent.com/13420573/56091645-b7f4b200-5eb1-11e9-8b78-56bad01d5f3b.png)